packages/ui/src/components/AccessibilityGuide.md

```diff
@@ -0,0 +1,78 @@
+# 1. Accessibility guide
+
+We want to provide following accessibility features:
+- roles & aria attributes,
+- keyboard accessibility,
+- proper typography, colors and contrast ratio.
+
+**Source of truth:** https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles and https://www.w3.org/WAI/standards-guidelines/aria/
+
+## 1.1. Roles & aria attributes
+
+Every tag has its role and list of attributes related to it. For example, `<button />` tag has
+`button` role. If you are forced to use an unsuitable tag (`div` as `button`) you can append the proper role, so it will be equal to the tag `button`:
+
+`<div role="button" />`
+
+**Role doesn't influence visual state of the tag. It's used only by screen-readers and has no effect for regular browsers.**
+
+As mentioned, every role has a list of attributes, related to it. Some of them specific to the role, others - common for all (or almost for all).
+
+For example, `button` role has specific attributes:
+- `aria-pressed`,
+- `aria-expanded` etc.
+
+And some common:
+- `aria-label`,
+- `aria-labeledby`,
+- `aria-disabled` etc.
+
+Try to append as much aria-attributes as possible, because it will give the additional information for screen-readers.
+
+**How can you understand, which roles & aria-attributes belong to the one or another block?** Human logic, source of truth and Google (`okey, google, tabs accessibility`).
+
+Some other ground rules:
+- pure graphics (like icons) which aren't used as button or link may be hidden via `aria-hidden` attribute;
+- all images should have `alt` attribute, even it's empty, **very important**;
+- all links must contain text, describing where user will be taken by them (home page, cart etc.). In case of images-links use `aria-label` attribute.
```

Comment:
```suggestion
- all links must contain text, which describes where user will be taken by them (home page, cart etc.). In case of images-links use `aria-label` attribute.
```